.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: scala
+scala:
+- 2.11.8
+jdk:
+- oraclejdk8
+script:
+- sbt test
+before_deploy:
+- pip install --user release-manager==0.2.0
```

Comment:
The indents are always messed up after adding credentials.  I normally re-indent to make it more readable but it all still works.